README.md

```diff
@@ -1,5 +1,5 @@
 # DataTable
-[![Build Status](https://secure.travis-ci.org/veracross/data-table.svg)](http://travis-ci.org/veracross/data-table)
```

Comment:
It is worth replacing this with a CircleCI status badge? https://circleci.com/docs/2.0/status-badges/

Reply:
Sure, why not